### PR TITLE
👑count moves and show all pretty

### PR DESCRIPTION
**Pull request by notwaldorf**
- :trophy: I added a move/time counter which is mostly always correct
- then I stole the style from your AB game to display it all pretty. Feel free to nuke the font and the colours and the words but if you add back the alert then :sob:  
- and then I fixed a typo and an unclosed `<span>` because I am mad with power

screen shot 2015-05-24 at 11 30 09 am
screen shot 2015-05-24 at 11 29 54 am

### CODE REVIEW

**Comment by muan** [comment]
LOLOLOLOL all the neightbors

File: index.html [hunk 1]
```diff
@@ -110,10 +128,13 @@
         game.restart(true)
       })
     </script>
+
+    <div id="outcome" style="display: block;"></div>
     <br>
     <span class="btn customize">
       CUSTOMIZE
       <code>new Game(cols, rows, bombs, [emptyemoji, bombemoji, flagemoji, starteremoji], twemojiOrNot)</code>
     </a>
```
Comment (muan) [inline index.html]: ^ heh this can be deleted. The `<span>` was originally `<a>` which was why it isn't closed. :laughing:

**Comment by notwaldorf** [comment]
Ughhhhhh this double counts sometimes

File: index.html (hunk 1 above)
Comment (notwaldorf) [inline index.html]: Delete the whole `customize` button thingie, you mean?

**Comment by notwaldorf** [comment]
Ohhhhhh it's because it sets a new event listener on every new game. Fixing! :D

File: index.html (hunk 1 above)
Comment (muan) [inline index.html]: Oh noes, I meant the `</a>` can be deleted, it should've been `</span>`.

Comment (muan) [inline index.html]: Welp actually I don't mind that it's gone. :+1:

**Comment by muan** [comment]
:sparkles::sparkles::sparkles::sparkles::sparkles::sparkles::sparkles::sparkles::sparkles::sparkles::sparkles::sparkles: 

> then I stole the style from your AB game to display it all pretty. Feel free to nuke the font and the colours and the words but if you add back the alert then :sob:

:laughing: I nuked the font and the colors and the words, but no alerts!!!! check this out: :heart_eyes: 

image

Reference: :laughing: 

image

Thank you!!!

---

I also changed some of the behaviours.
- flagging is not a move
- double click (is a new thing) is one move not 2 moves.
- timer only start at first click :space_invader: 

The game looks pretty serious now! perhaps there should be a tournament :bangbang:

**Comment by notwaldorf** [comment]
Eeeeeeee you made it so pretty! You're the best! ✨👏

**Pull request merged**